src/Extensions/CacheKeyExtension.php

```diff
@@ -28,37 +28,13 @@ class CacheKeyExtension extends DataExtension
 
     public function updateCMSFields(FieldList $fields): void
     {
-        if (!$this->owner->config()->get('remove-cache-keys-field')) {
-            return;
-        }
-
-        $fields->removeByName('CacheKeys');
-    }
-
-    public function findCacheKeyHash(): ?string
-    {
-        if (!$this->owner->isInDB()) {
-            return null;
-        }
-
-        $hasCacheKey = $this->owner->config()->get('has_cache_key');
-
-        if (!$hasCacheKey) {
-            return null;
+        if ($this->owner->config()->get('remove-cache-keys-field')) {
+            $fields->removeByName('CacheKeys');
```

Comment:
I feel like we should check for when people want to add the field rather than remove it as my understanding would be to hide it by default

Reply:
Actually, could this sit in say the settings tab as a literal field so devs can see it for debugging purposes?

Reply:
Thanks @adrhumphreys!

At the moment it shows up as a tab with a gridfield, but yea, I like the idea of making the field a bit more simple and "display only" in the Settings area.

Also, sweet. I'll go with hidden by default 😃